assets/scss/components/_card.scss

```diff
@@ -0,0 +1,61 @@
+.c-card__title {
+    color: $color-black;
+    font-family: $font-semi-bold;
+    font-size: 1.125rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5rem;
+    text-decoration: none;
+}
+
+.c-card__product-information {
+    display: flex;
+    flex-direction: column;
+    gap: 0.75rem;
+}
+
+.c-card__figure {
+    border-radius: 0.25rem;
+    margin-bottom: 0.75rem;
+    position: relative;
+    overflow: hidden;
+}
+
+.c-card__image {
```

Comment:
@mgbelegu, don't you think that `c-card__image-wrapper` suits better here? And then, instead of nesting the `<img>`, we could use `c-card__image` for the image itself.